docker-stack.yml

```diff
@@ -3,6 +3,8 @@ version: '3'
 services:
   frankyapp:
     image: franky_frankyapp:latest
+    env_file:
+      - .env
```

Comment:
I didn't noticed it before but why is there a need for both `docker-compose.yml` and `docker-stack.yml` files? I know nothing about the docker stack technology but according to the [docs](https://docs.docker.com/get-started/part5/) `docker-compose.yml` can be used with stack operations at least at some cases.

Probably, we can merge  `docker-compose.yml` and `docker-stack.yml` into a single file. What do you think?

Reply:
+++